outputs.tf

```diff
@@ -44,8 +44,8 @@ output "static_ip" {
   description = "The external (public) static IPv4 for the Domino UI"
 }
 
-output "uuid" {
-  value       = local.uuid
+output "deployment_id" {
```

Comment:
so this one is a bit tricky... 
it doesn't match the description and you will have to make additional changes on platform-apps to alight with the new output name.